src/amo/components/Collection/index.js

```diff
@@ -149,15 +144,19 @@ export class CollectionBase extends React.Component<Props> {
   }
 
   editCollectionLink() {
-    const { _config, i18n } = this.props;
+    const { _config, i18n, location } = this.props;
     const props = {};
 
+    const pageQueryParam =
+      location.query.page ? `?page=${location.query.page}` : '';
+    const editUrl = `${this.editUrl()}${pageQueryParam}`;
```

Comment:
I'm not sure. I suppose it would protect us from the implementation of the query param changing, but right now both the filter name and the query param are `page`, so it doesn't really buy us much. I guess it doesn't hurt either, so I'll do it.

Reply:
Hmm, actually, `convertFiltersToQueryParams` returns an object to us (I had thought it might have returned a string), so if I use it, I then have to translate from the object to a string, because I can't assume the object will have a key for `page`, because that kind of defeats the point of using `convertFiltersToQueryParams` in the first place.  I think the resulting code would be longer, and more complex, than what I have right now, would be which is actually quite simple. So I think I'd prefer to keep it as is. What do you think, @willdurand?